testsuite/MDAnalysisTests/analysis/test_rdf.py

```diff
@@ -94,6 +94,21 @@ def test_exclusion(sels):
     assert rdf.results.count.sum() == 4
 
 
+@pytest.mark.parametrize("attr", ("residue"))
```

Comment:
```suggestion
@pytest.mark.parametrize("attr", ("residue",))
```
A one-element tuple needs the final `,`. But probably best to check all the attributes as suggested in https://github.com/MDAnalysis/mdanalysis/pull/4161#discussion_r1237891018. @hmacdope do you know of appropriate existing test data for segindices and chainids? Otherwise it should be possible to create something *ad-hoc* within the test.